web-local/src/lib/components/dimension/DimensionTable.svelte

```diff
@@ -210,6 +220,7 @@ TableCells – the cell contents.
 >
   <div
     bind:this={container}
+    on:scroll={() => (horizontalScrolling = container?.scrollLeft > 0)}
```

Comment:
For the sake of legibility

```suggestion
    on:scroll={() => { horizontalScrolling = container?.scrollLeft > 0 }}
```